tests/io/eager/test_read.py

```diff
@@ -26,7 +26,8 @@ def output_type(request):
 
 def _read_table(*args, **kwargs):
     if "tables" in kwargs:
-        kwargs.pop("tables")
+        param_tables = kwargs.pop("tables")
+        kwargs["table"] = param_tables
```

Comment:
same

Reply:
I am adding `table` here instead of `tables`. If the `table` is not provided the test case will fail for `read_table` in eager.
Also, I need to write a common test case that would be applicable to all the backends. 

https://github.com/JDASoftwareGroup/kartothek/blob/785ec8a1a54aca13f2e7cb076b970ddd0ebb275e/kartothek/io/eager.py#L300